distribution/docs/src/main/resources/_contents/_securing/catalog-filtering-contents.adoc

```diff
@@ -0,0 +1,129 @@
+
+==== Catalog Filtering
+
+Filtering is performed by an Access plugin, after a query or delete has been performed or before ingest has been performed.
+
+===== How Filtering Works
+
+Each metacard result can contain security attributes that are pulled from the metadata record after being processed by a `PolicyPlugin` that populates this attribute.
+The security attribute is a Map containing a set of keys that map to lists of values.
+The metacard is then processed by a filter plugin that creates a `KeyValueCollectionPermission` from the metacard's security attribute.
+This permission is then checked against the user subject to determine if the subject has the correct claims to view that metacard.
+The decision to filter the metacard eventually relies on the installed PDP (`feature:install security-pdp-authz`).
```

Comment:
Not sure if it matters in this context but wondering if we should spell out Policy Decision Point the first time here.